Makefile

```diff
@@ -1,36 +1,37 @@
 SHELL := /bin/bash
-WHEELHOUSE := ./wheelhouse
 PYVER := 3.6
 VENV := ./env$(PYVER)
 
-version = $(shell "bin/get-version.sh")
+version := $(shell "bin/get-version.sh")
 
-dist/spacy-$(version).pex : wheelhouse/spacy-$(version)-*.whl
-	pex -f ./wheelhouse --no-index --disable-cache -m spacy -o dist/spacy-$(version).pex spacy==$(version) jsonschema
-	chmod a+rx dist/spacy-$(version).pex
+dist/spacy-$(version).pex : wheelhouse/spacy-$(version).stamp
+	$(VENV)/bin/pex -f ./wheelhouse --no-index --disable-cache -m spacy -o $@ spacy==$(version) jsonschema
```

Comment:
Ah not running pex within the virtualenv was was definitely an oversight. Nice idea with the timestamp too.

Reply:
(Officemates who are better at Makefiles are helpful.)